front/src/assets/rollingStock/freightRollingStocks.ts

```diff
@@ -108,6 +108,11 @@ const FREIGHT_ROLLING_STOCKS = [
   'Y8000AG',
   'Y8000AP',
   'Y9000US',
+  'electric_rolling_stock_test_e2e',
+  'dual-mode_rolling_stock_test_e2e',
+  'slow_rolling_stock_test_e2e',
+  'fast_rolling_stock_test_e2e',
+  'improbable_rolling_stock_test_e2e',
```

Comment:
those rollingstocks are only used for tests

in order to separate the usecases, can't we declare them in an asset file in `tests/` then spread them in this array only if the tests are started?

we could use `fs` at global setup to create the array to spread, this array will be empty if the tests are not lauched

Reply:
The list of rolling stocks in `freightRollingStocks.ts` is a temporary solution for the project, not just for the tests. I believe we can leave it as is until the underlying logic is updated, at which point we can adjust the tests accordingly

Reply:
ok, a comment would have been good to explain it, but it's not related to your issue